receiver/mysqlreceiver/documentation.md

```diff
@@ -21,7 +24,7 @@ These are the metrics available for this scraper.
 | **mysql.row_locks** | The number of InndoDB row locks. | 1 | Sum(Int) | <ul> <li>row_locks</li> </ul> |
 | **mysql.row_operations** | The number of InndoDB row operations. | 1 | Sum(Int) | <ul> <li>row_operations</li> </ul> |
 | **mysql.sorts** | The number of MySQL sorts. | 1 | Sum(Int) | <ul> <li>sorts</li> </ul> |
-| **mysql.threads** | The state of MySQL threads. | 1 | Sum(Double) | <ul> <li>threads</li> </ul> |
+| **mysql.threads** | The state of MySQL threads. | 1 | Sum(Int) | <ul> <li>threads</li> </ul> |
```

Comment:
BTW, I believe at some point we might want to update the [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions) with database metrics. I guess this will also bring all sort of questions (e.g. if some of them should be named per database or be more generic, so `mysql.threads` or `dbms.threads` or so)

There is already a somewhat related proposal on [client connection pool](https://github.com/open-telemetry/opentelemetry-specification/pull/2273/files)

Reply:
I agree, it's going to be increasingly important to solidify all of this in the semantic conventions. In the short term, I've found it useful to work through a lot of the finer details against an actual implementation. However, I'm sure there will be some updates to these metrics once they've been more broadly scrutinized.